### PR TITLE
fix(#150): add maxlength on input

### PR DESCRIPTION
**Pull request by jufab**
#### Summary

Add a Max Length on user input.
Max length = 22

#### Ticket Link

Fixe #150

### CODE REVIEW

**Review by mickmister — COMMENTED** [review]
Thanks for your contribution @jufab :tada:

Mostly looks good, I have just one request to use type inference to avoid the use of `@ts-ignore` plus a few small comments. Otherwise LGTM :+1:

**Comment by mickmister** [comment]
Also there are a few linting issues being reported by CI:
https://github.com/mattermost/mattermost-plugin-todo/actions/runs/6581802885/job/17902038384?pr=222#step:4:8620

```
Error:    19:1   error  `react-select` import should occur before import of `../../utils`  import/order
Error:    34:1   error  More than 1 blank line not allowed                                 no-multiple-empty-lines
Error:   122:20  error  'props' is already declared in the upper scope                     no-shadow
Error:   122:59  error  Prop `maxLength` must be placed on a new line                      react/jsx-max-props-per-line
Error:   122:74  error  A space is forbidden before closing bracket                        react/jsx-tag-spacing
```

Can you please take a look at these errors @jufab? Thank you!

File: webapp/src/components/user_selector/autocomplete_selector.tsx [hunk 1]
```diff
@@ -116,6 +118,9 @@ export default function AutocompleteSelector(props: Props) {
         );
     }
 
+    //@ts-ignore
+    const Input = (props) => <components.Input {...props} maxLength={22} />;
```
[the remark this answers is not in the document]
Comment (jufab) [reply webapp/src/components/user_selector/autocomplete_selector.tsx:122]: If I inline this code, I have an other problem. 😢 

image

[the remark this answers is not in the document]
Comment (mickmister) [reply webapp/src/components/user_selector/autocomplete_selector.tsx]: Okay then it seems the types for the library don't agree with its supported functionality. Yeah let's just use `@ts-ignore` here. Thanks for trying this :+1:

**Review by mickmister — APPROVED** [review]

**Review by larkox — APPROVED** [review]

**Review by AayushChaudhary0001 — APPROVED** [review]
Tested this PR for the following scenarios:-
- Username field can accept maximum length upto 22 characters while creating a ToDo.
- Username field can accept maximum length upto 22 characters while assigning a ToDo.

The PR was found working fine for both the above conditions, LGTM. Approved.

**Comment by mickmister** [comment]
Thanks @jufab!!

**Pull request merged**